src/providers/formattingProvider.ts

```diff
@@ -66,8 +71,9 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
             const formattedLine = originalLine
                 .replace(/^\s*/, '')
                 .replace(/;.+/, '')
-                .replace(/ {2,}/g, ' ')
+                .replace(/\s{2,}/g, ' ')
```

Comment:
If the user indents with tabs, won't this replace tabs with spaces? I think we should leave this behavior as-is please :) we can address it in a separate PR if necessary.

Reply:
Done